### ISSUE TITLE
FR: Use a markdown formatter for consistency

### ISSUE THREAD

**Issue by SkepticMystic** [obsidian-community/obsidian-hub#31]
I propose that we use a formatter to keep things consistent.
I personally use Prettifier, but we could totally use a different one. We can put a Prettifier config file in the vault so that everyone uses the same settings

**Comment by konhi** [comment]
Cool idea! If we don't want to make it harder for contributors, auto-formatting with GitHub Actions could be done as well.

**Comment by SkepticMystic** [comment]
Good idea! I'm fine with either. We can point to the same Prettifier config file, so that users can format notes while editing, too

**Comment by claremacrae** [comment]
A note for when we turn on auto-reformatting of markdown files for consistency.

We need to make sure we opt for making `-` the chosen list character, not `*`

### Background

In #553 we have some code for sorting lists of plugins by plug-in.
All the plugin lists currently use `-` as the list character, and that is what the script supports.

If we were to choose `*`, it would break the sorting of plugin lists.
